backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/notification/NotificationSenderTemplate.java

```diff
@@ -209,6 +211,20 @@ else if (recipient.isAllRolesContainingGroup())
 							.stream()
 							.map(userId -> Recipient.userAndRole(userId, roleId)));
 		}
+		else if (recipient.isOrgUsersContainingGroup())
+		{
+			final List<UserId> userIds = userNotificationsConfigRepository.getByNotificationGroupAndOrgId(recipient.getNotificationGroupName(), recipient.getOrgId());
+			if (userIds.isEmpty())
+			{
+				final UserId deadletterUserId = notificationGroupNamesRepo.getDeadletterUserId(recipient.getNotificationGroupName());
```

Comment:
IMHO the dead-letter rule shall be applied in all cases where we don't get a recipient to send too

Reply:
added for `AllRolesContainingGroup` as well (it's the only other possible type, because we need a Notification Group Name).